pyproject.toml

```diff
@@ -19,6 +19,7 @@ attrs = "^21.4.0"
 pycooldown = "^0.1.0-beta.6"
 croniter = "^1.3.5"
 types-croniter = "^1.0.10"
+black = "^22.3.0"
```

Comment:
this should be a dev dependency

Reply:
There was something broken with black and I must have reinstalled it incorrectly. Will fix.